### PR TITLE
fix lyrics command

### PR DESCRIPTION
**Pull request by LBlend**
Lyrics command should now work.

That does not mean it is working well due to the limitations of the genius API, however it does have >0% success rate now :P
It isn't efficent either since I have to scrape the data, though the code should be a little bit cleaner.

I'm suggesting merging this for now and rewrite it when (if) a better API comes along.

**Note**:
Translations to `we_eb` and `nn_no` are missing.

### CODE REVIEW

File: musicbot/cogs/music/__init__.py [hunk 1]
```diff
@@ -635,69 +634,104 @@ async def _history(self, ctx):
     @commands.cooldown(1, 5, commands.BucketType.guild)
     @commands.command(name='lyrics')
     async def _lyrics(self, ctx, *query: str):
+        """ Search for lyrics of a song """
+        # Check for API key
+        if not (genius_access_token := self.bot.APIkeys.get('genius')):
+            return await ctx.send(ctx.localizer.format_str('{history.title}'))
```
Comment (Ev-1) [inline musicbot/cogs/music/__init__.py]: history.title?

**Review by Ev-1 — COMMENTED** [review]
Looks good to me, just wondering about the one localization command.

File: musicbot/cogs/music/__init__.py (hunk 1 above)
Comment (Ev-1) [reply musicbot/cogs/music/__init__.py]: Should be `errors.missing_api_key`

Comment (LBlend) [reply musicbot/cogs/music/__init__.py]: oops. copilot probably got the best of me. fixed

**Review by Ev-1 — APPROVED** [review]
lgtm

**Pull request merged**